tls/s2n_crypto.h

```diff
@@ -69,7 +70,9 @@ struct s2n_crypto_parameters {
     struct s2n_pkey client_public_key;
     struct s2n_dh_params server_dh_params;
     struct s2n_ecc_params server_ecc_params;
+    struct s2n_ecc_evp_params server_ecc_evp_params;
```

Comment:
seems like the addition in this files aren't used yet. could we differ this change to the next PR?

Reply:
will break this PR up.